### PR TITLE
(TRAINTECH-1389) Hide Terminal When Logged Out

### PR DESCRIPTION
**Pull request by marrero984**
TRAINTECH-1389 #resolved #time 1h
When no student user is logged in, hide the webterminal and display
 instructions on how to proceed.  This occurs even if the admin is
 logged in.

Admin view while not logged in as a user:
screen shot 2017-10-27 at 11 52 57 am

### CODE REVIEW

**Comment by binford2k** [comment]
What's that first image?

**Comment by binford2k** [comment]
Can you put an `<hr>` between the two forms, or some kind of border or some visual separation?

**Comment by marrero984** [comment]
Below 1024px.  Working on fixing the menu now

File: views/shell.erb [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-    </ul>
-  </div>
+  <p>
+    If you prefer, you may also use any standard SSH client using the username and password you specified. Some common SSH clients include:
+  </p>
+  <ul>
+    <li>
+      Mac OSX and Linux:
+      <ul>
+        <li>Terminal.app</li>
```
Comment (binford2k) [inline views/shell.erb]: These are not SSH clients. The original wording was more accurate, though adding links or descriptions of the terminals is probably a good idea.

**Comment by marrero984** [comment]
@binford2k Updated

**Pull request merged**